webpack/envs/commonEnv.js

```diff
@@ -6,6 +6,8 @@ import { basePath, env } from "../utils/env"
 
 export const standardDevtool = env.webpackDevtool || "eval"
```

Comment:
@icirellik, @joeyAghion - per this thread: https://artsy.slack.com/archives/CA8SANW3W/p1619549886353000?thread_ts=1619549416.352400&cid=CA8SANW3W

I have this overwritten locally via `.env`, but note that Chrome doesn't like `eval` source maps any more (not sure why, or how to fix), and in any case we should use `cheap-module-eval-source-map` for this setting so that the team gets real uncompiled typescript / coffeescript code in chrome devtools, rather than compiled code. This setting is fast.